onnx/test/shape_inference_test.py

```diff
@@ -2311,19 +2311,21 @@ def test_gemm_no_bias(self) -> None:
 
     def test_reduce_op_shape_2_axis(self) -> None:
         graph = self._make_graph(
-            [("x", TensorProto.FLOAT, (24, 4, 11))],
-            [make_node("ReduceL1", "x", "y", axes=(1, 2), keepdims=0)],
+            [("x", TensorProto.FLOAT, (24, 4, 11)), ("axes", TensorProto.INT64, (2,))],
```

Comment:
Should we be retaining the old test-cases for the old version, and add copies of the test-cases for the new version?

Reply:
brought back the tests for opset 13. added tests for opset 18